doc/theory/glossary.tex

```diff
@@ -162,6 +154,31 @@
 	
 	A \textit{cellwise filtration} is a complex $X$ together with a total order $\leq$ on its simplices or elementary cubes such that for each $y \in X$ the set $\{x \in X\ :\ x \leq y\}$ is a subcomplex of $X$. A cellwise filtration can be naturally thought of as a \hyperref[filtered complex]{filtered complex}.
 	
+	\subsection*{Clique and flag complexes} \label{clique and flag complexes}
+	
+	Let $G$ be a $1$-dimensional abstract (resp. directed) simplicial complex. The abstract (resp. directed) simplicial complex $\langle G \rangle$ has the same set of vertices as $G$ and $\{v_0, \dots, v_n\}$ \big(resp. $(v_0, \dots, v_n)$\big) is a simplex in $\langle G \rangle$ if an only if $\{v_i, v_j\}$ \big(resp. $(v_i, v_j)$\big) is in $G$ for each pair of vertices $v_i, v_j$. 
+	
+	An abstract (resp. directed) simplicial complex $X$ is a \textit{clique (resp. flag) complex} if $X = \langle G \rangle$ for some $G$.
+	
+	Given a function
+	\begin{equation*}
+	w : G \to \mathbb R \cup \{\infty\}
+	\end{equation*}
+	consider the extension
+	\begin{equation*}
+	w : \langle G \rangle \to \mathbb R \cup \{\infty\}
+	\end{equation*}
+	given by
+	\begin{equation*}
+	w(\sigma) = \max(\{ w(e)\ |\ e \in G \cap \sigma\})
```

Comment:
But I wonder if the intersection is valid? Since \sigma is an element of G?

Reply:
You are right, I am updating with an alternative.